packages/blade/src/components/Button/BaseButton/BaseButton.stories.internal.tsx

```diff
@@ -9,7 +9,6 @@ export default {
   component: BaseButtonComponent,
   args: {
     variant: 'primary',
-    contrast: 'low',
```

Comment:
Because we have added a restriction that you cannot use contrast along with variant. To use contrast & intent, you shouldn't use variant. Those combinations are not supported through design